bluebutton-data-pipeline-rif/src/main/java/gov/hhs/cms/bluebutton/datapipeline/rif/model/CompoundCode.java

```diff
@@ -0,0 +1,53 @@
+package gov.hhs.cms.bluebutton.datapipeline.rif.model;
+
+/**
+ * Enumerate the possibly values for Compound Code for Part D
+ */
+public enum CompoundCode {
+	NOT_COMPOUNDED(1, "Not Compounded"), COMPOUNDED(2, "Compounded");
+
+	private final Integer code;
+	private final String description;
```

Comment:
I'm not sure that it would be ever used, so I don't necessarily have a strong preference for keeping it here.